### ISSUE TITLE
[Dropdown] Multiple dropdown doesn't show selected label if the selected value has ampersand "&"

### ISSUE THREAD

**Issue by gfkateo** [fomantic/Fomantic-UI#2782]
# Bug Report
When creating a multiple dropdown using `<select>` element and the selected option's value contains ampersand, the label for the selected value doesn't show.

## Expected result
Should show the label of the selected option 

![Screenshot from 2023-05-09 16-33-50](https://user-images.githubusercontent.com/100657275/237040985-2e852575-3bf0-4c26-b4d8-31ae413b2923.png)

## Actual result
 
![Screenshot from 2023-05-09 16-21-34](https://user-images.githubusercontent.com/100657275/237037922-f998d53c-e6b0-41d4-9264-f593c53f1175.png)

Ran this code in the console
```javascript
$('.ui.dropdown.sample1').dropdown('get value')
```
and it returns the correct value
```
["b & c"]
```

Also, I've tested creating a dropdown with using `<div class="ui multiple selection dropdown">` element, and it doesn't have this issue.  (See Sample 3 field in the Codepen testcase)

## Testcase
[Codepen URL](https://codepen.io/katedo/pen/ExdENQW)

### HTML
```html
<div class="field">
  <label>Sample 1: Using select element with ampersand</label>
  <select name="values1" class="ui fluid dropdown sample1" multiple>
    <option value="">Select at least one</option>
    <option value="a">A</option>
    <option value="b & c" selected>B & C</option>
    <option value="d">D</option>
  </select>
</div>
```

### Javascript
```javascript
$('.ui.dropdown').dropdown();
```

## Version

2.9.2

**Comment by ko2in** [comment]
I can't reproduce that problem in your codepen.

![fui-ampersand-in-dropdown](https://github.com/fomantic/Fomantic-UI/assets/930315/3f7e84ec-be38-4914-b0bf-c475ab5d6192)


What is your browser and it's version?

**Comment by lubber-de** [comment]
@ko2in The issue is, that the entry containing the ampersand has got the "selected" attribute set. So it should be preselected / and the label should be shown when the codepen is run (without selecting anything)

**Comment by ko2in** [comment]
Oh I see. I tried with encoded characters (```&amp```;, ```&#x26```, ```&#38```) and still have the problem. Native select element doesn't have that issue.

**Comment by lubber-de** [comment]
Fixed by #2790 
See https://jsfiddle.net/lubber/qykf58tc/2/
